cmd/kube-proxy/app/server.go

```diff
@@ -197,7 +197,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 		"This parameter is ignored if a config file is specified by --config.")
 
 	fs.StringSliceVar(&o.config.NodePortAddresses, "nodeport-addresses", o.config.NodePortAddresses,
-		"A list of CIDR ranges that contain valid node IPs. If set, connections to NodePort services will only be accepted on node IPs in one of the indicated ranges. If unset, NodePort connections will be accepted on all local IPs. This parameter is ignored if a config file is specified by --config.")
+		"A list of CIDR ranges that contain valid node IPs. If set, connections to NodePort services will only be accepted on node IPs in one of the indicated ranges. If this is unset and --nodeport-addresses-primary is true, then NodePort connections will be accepted only on the node's primary IP(s). If this is unset and --nodeport-addresses-primary is false, then NodePort connections will be accepted on all local IPs. This parameter is ignored if a config file is specified by --config.")
+	fs.BoolVar(&o.config.NodePortAddressesPrimary, "nodeport-addresses-primary", o.config.NodePortAddressesPrimary, "Indicates that NodePort service connections should only be accepted on the node's primary IP(s) as indicated by the Node object. This must be false if --node-port-addresses is non-empty. This parameter is ignored if a config file is specified by --config.")
```

Comment:
Should we need to indicate that this option is set to `true` by default in `nftables` mode?

Reply:
I think while nftables is still alpha and requires a feature gate to be set, we shouldn't mention it in the other options... I guess I could add a feature gate check here...

Reply:
I decided against this for now because it makes the doc string too complicated...